comfy_cli/command/custom_nodes/command.py

```
@@ -663,19 +668,58 @@ def install(
+        code = normalize_cm_cli_exit_code(raw)
+        failed_argv = e.cmd if isinstance(e.cmd, list | tuple) else [e.cmd]
+        ran_cm_cli = any(str(part).replace("-", "_") == "cm_cli" for part in failed_argv)
+        if fast_deps and not ran_cm_cli:
+            # With --fast-deps, execute_cm_cli runs DependencyCompiler after cm-cli
+            # succeeds; a pip/uv failure there is not a cm-cli failure — the packs
+            # themselves installed.
+            message = (
+                f"dependency installation after `cm-cli install` failed with exit code {raw} "
+                "(the packs installed; their dependencies did not)."
+            )
+            hint = "see the pip/uv output above for the failing dependency; re-run `comfy node install --fast-deps --exit-on-fail ...` to retry"
```

Pylint (4.0.6)

[convention] Line too long (144/120)

(C0301)